mailpoet/mailpoet
# Add Product Discovery section to homepage [MAILPOET-4827]

## Description
This PR adds the product discovery section to the homepage
<img width="757" alt="Screenshot 2023-01-12 at 16 32 19" src="https://user-images.githubusercontent.com/1082140/212110031-e245a623-a816-4d6a-bfe9-be30a16e2e49.png">


## Code review notes
I recommend reviewing commit by commit.

## QA notes
When testing using the ZIP build, the illustration images in the task list won't load because they will be publicly available after the release.
You can check the feature with the images by testing with the dev env and adding `define( 'MAILPOET_USE_CDN', false ); ` to the wp-config.php. Alternatively, you can ping me and I'll push the images to the repo so that they also work from the zip.

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-4827]

## After-merge notes

_N/A_


[MAILPOET-4827]: https://mailpoet.atlassian.net/browse/MAILPOET-4827?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
